tests/graphql/transaction.spec.ts

```diff
@@ -257,6 +257,28 @@ describe("Transaction", () => {
             name_likeAny: ["1234", "-567", "86.12", "90,1"],
             operator: BaseOperator.Or,
             purpose_likeAny: ["1234", "-567", "86.12", "90,1"],
+            conditions: [
```

Comment:
Existing test cases cover all of the new filter cases, so I didn't add new ones specifically for this.